articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -120,28 +111,23 @@ attributes.
 > Where:
 >
 > - `Enter_the_Application_Id_here` - is the Application Id for the application you registered.
-> - `Enter_the_Client_Secret_Here` - is the **Client Secret** you created in **Certificates & Secrets** for the application
-you registered.
+> - `Enter_the_Client_Secret_Here` - is the **Client Secret** you created in **Certificates & Secrets** for the application you registered.
 
 #### Step 4: Run the code sample
 
 To run the project, you can either:
 
-- Run it directly from your IDE by using the embedded spring boot server
-- or package it to a WAR file using [maven](https://maven.apache.org/plugins/maven-war-plugin/usage.html) and deploy it to a J2EE container solution such as [Apache Tomcat](http://tomcat.apache.org/).
+- Run it directly from your IDE by using the embedded spring boot server or
+- Package it to a WAR file using [maven](https://maven.apache.org/plugins/maven-war-plugin/usage.html) and deploy it to a J2EE container solution such as [Apache Tomcat](http://tomcat.apache.org/).
```

Comment:
This is a little awkward with the sentence fragment. Here are some options. Choose one that you like:
- Delete the trailing 'or' and add a period after 'server'
- Write it as one sentence and git rid of the leading '-'

Reply:
Writing it as one sentence